quarkus/src/main/resources/application.yml

```diff
@@ -38,23 +38,22 @@ quarkus:
   quarkus:
     #synchronus datasource settings, should be used only for initializing local database
     datasource:
-      jdbc: false
       ## Once Flyway create database comment below jdbc configuration:
-      reactive:
+      jdbc:
```

Comment:
If that didn't work for you, it would be good to comment it and add new, changed line.